src/apps/lwaftr/binding_table.lua

```diff
@@ -158,9 +155,9 @@ function BindingTable:remove_softwire_entry(entry_key_blob)
    self.softwires:remove(entry.key)
 end
 
-local lookup_key = softwire_key_t()
 function BindingTable:lookup(ipv4, port)
-   local psid = self:lookup_psid(ipv4, port)
+   local lookup_key = ffi.new(self.softwires.entry_type).key
```

Comment:
This is going to be bad for lookup perf (though we usually use the other interface).  Can we add a FIXME here?